modules/rostests/apitests/user32/WndProc.c

```diff
@@ -9,96 +9,143 @@
 
 /* Used wine Redraw test for proof in principle. */
 
+#define WMPAINT_COUNT_THRESHOLD 10
+
 /* Global variables to trigger exit from loop */
 static int redrawComplete, WMPAINT_count;
 
 /*
-   Force stack corruption when calling from assumed window procedure callback.
-   Adding (6 and) more will force exception faults and terminate the test program.
-   The test is with five and this is safe for windows.
-
-   But,,,, ReactOS compiled with GCC can handle this,,,,,,
+ * Force stack corruption when calling from assumed window procedure callback.
+ * Adding (6 and) more will force exception faults and terminate the test program.
+ * The test is with five and this is safe for windows.
+ *
+ * But,,,, ReactOS compiled with GCC can handle this,,,,,,
  */
-static LRESULT WINAPI redraw_window_procA(HWND hwnd, UINT msg, WPARAM wparam, LPARAM lparam, DWORD ExtraData,DWORD ExtraData1,DWORD ExtraData2,DWORD ExtraData3)
+static LRESULT WINAPI redraw_window_procA(
+    HWND hwnd,
+    UINT msg,
+    WPARAM wparam,
+    LPARAM lparam,
+    DWORD ExtraData,
+    DWORD ExtraData1,
+    DWORD ExtraData2,
+    DWORD ExtraData3)
 {
     switch (msg)
     {
     case WM_PAINT:
-        trace("doing WM_PAINT %d\n", WMPAINT_count);
         WMPAINT_count++;
-        if (WMPAINT_count > 10 && redrawComplete == 0)
+        trace("Doing WM_PAINT %d/%d\n", WMPAINT_count, WMPAINT_COUNT_THRESHOLD);
+
+        if (WMPAINT_count > WMPAINT_COUNT_THRESHOLD && redrawComplete == 0)
         {
             PAINTSTRUCT ps;
+
+            trace("Calling *Paint()\n");
             BeginPaint(hwnd, &ps);
             EndPaint(hwnd, &ps);
             return 1;
         }
-   /*
-       This will force one stack corruption "ret" fault with normal window
-       procedure callback.
-    */
+
+        /*
+         * This will force one stack corruption "ret" fault with normal window
```

Comment:
Is this a new comment style?

Reply:
Hum. You wanted `//`s, right?

Reply:
No, but adding '*' in front of all lines is not needed at all.